truss/_integration-tests/README.md

```diff
@@ -0,0 +1,38 @@
+# integration-tests
+
+These tests run truss against definition files and tests if the generated code
+behaves. 
+
+# ./http
+
+The http directory contains http transport tests. They include tests against
+the generated http transport client libraries against the server and testing
```

Comment:
I think this is intended to read:

> They include tests ~~against~~ **of** the generated http transport client libraries against the server**,** and testing hand-made HTTP requests against the server.

In general, remember to capitalize HTTP.